### PR TITLE
Plane: Add missing  Q_OPTIONS metadata for all options

### PR DESCRIPTION
**Pull request by Hwurzburg**
added missing  descriptions...made format more consistent

### CODE REVIEW

File: ArduPlane/quadplane.cpp [hunk 1]
```diff
@@ -328,8 +328,8 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
...
+    // @Bitmask: 0:LevelTransition,1:AllowFWTakeoff,2:AllowFWLand,3:Respect Takeoff Frame,4:Use Fixed Wing Approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use Idle Governor in MANUAL,7:QAssist Force enabled,8:Tailsitter QAssist motors only,9:Enable AirMode if arming by aux switch,10:Enable Motor Tilt when disarmed,11:Delay Spoolup for 2 seconds after arming
```
Comment (IamPete1) [inline ArduPlane/quadplane.cpp:332]: `Respect Takeoff Altitude Frame`, rest looks OK to me

Comment (Hwurzburg) [reply ArduPlane/quadplane.cpp:332]: oops , missed fixing one..thanks!

Comment (WickedShell) [reply ArduPlane/quadplane.cpp:332]: "Takeoff Frame" shouldn't be caps locked, it's not a proper noun/acronym. Same for "Fixed Wing Approach". Those were intentionally lower case.

Comment (Hwurzburg) [reply ArduPlane/quadplane.cpp:332]: the mixture of short name and acronyms is in consistent....perhaps they should all be acronyms for easy referral in the description...makes it clearer in the Wiki...
LevelTransition,AllowFWTakeoff,RespectTKOFrame,UseFWApproach,UseQRTL,UseGovernor,ForceQAssist,MtrsOnlyQAssist,AirModeEn,DelaySpoolUp ? they appear as a table...

Comment (WickedShell) [reply ArduPlane/quadplane.cpp:332]: I don't think acronyms would help honestly. There are already so many acronyms we all take for granted I think throwing more (very non standard) acronyms at people for this aren't going to help. In general I'd say anything that is short like `RespectTKOFrame` is actually user hostile. It's both not obvious to the reader what TKO is, and to make it worse it actively makes it harder for people who don't have english as their first language, as that type of thing is nearly impossible to get through a translation tool, or get a handle on with a dictionary, whereas spelling it out as `Respect the takeoff altitude frame` is quite clear in English, as well as having a chance to be used in other languages.

Comment (Hwurzburg) [reply ArduPlane/quadplane.cpp]: okay, then going the other way for consistency...no need for a long description if its in the param bitmask....checked MP and wiki and they would display correctly...

File: ArduPlane/quadplane.cpp [hunk 2]
```diff
@@ -328,8 +328,8 @@ const AP_Param::GroupInfo QuadPlane::var_info[] = {
...
-    // @Bitmask: 0:LevelTransition,1:AllowFWTakeoff,2:AllowFWLand,3:Respect takeoff frame types,4:Use a fixed wing approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use idle governor in MANUAL,7:QAssist force enabled,8:Tailsitter QAssist motors only,9:enable AirMode if arming by aux switch,10:Enable motor tilt when disarmed,11:Delay spoolup for 2 seconds after arming
+    // @Description: This provides a set of additional control options for quadplanes.
+    // @Bitmask: 0:Keep wings within LEVEL_ROLL_LIMIT and only use forward motor(s) for climb during transition,1:Allow fixed wing takeoff for TAKEOFF mission command,2:Allow fixed wing landing for LAND mission command,3:VTOL takeoff altitude is above current position rather than HOME,4:Use Fixed Wing Approach for VTOL landings,5:Use QRTL instead of QLAND for failsafe when in VTOL modes,6:Use Idle Governor in MANUAL for forward motor,7:Force QAssist on always,8:QAssist in tailsitters only uses VTOL motors,9:Enable AirMode if arming by aux switch,10:Enable motor tilt for yaw when disarmed,11:Delay VTOL spoolup for 2 seconds after arming
```
Comment (WickedShell) [inline ArduPlane/quadplane.cpp:332]: Lower case is better, but the description for bit 3 (the respect frame stuff) is now backwards. The default is the bit not set, in which case takeoffs are relative to the *current position* not home, or anything else. If you set the bit, then it follows the item you uploaded to the aircraft. The key point to understand here is if you plan a VTOL takeoff to 200 feet, but are at 400 feet and jump to the takeoff command the plane will takeoff another 200 feet to 600 feet above the ground if the bit isn't set.

Comment (Hwurzburg) [reply ArduPlane/quadplane.cpp:332]: will fix thanks...

**Comment by Hwurzburg** [comment]
@WickedShell good to go now?

**Comment by Hwurzburg** [comment]
@tridge okay, reworked as per DEV-CALL....hope you like the short names...

**Comment by tridge** [comment]
> @tridge okay, reworked as per DEV-CALL....hope you like the short names...

I'm find with them, but @WickedShell doesn't like them :-)
see his comment on #15407 which includes your patch

**Comment by Hwurzburg** [comment]
@tridge reworked again....you will need to rebase if this goes in first and fix the new option

**Comment by tridge** [comment]
@Hwurzburg looks good. I'll merge this when CI passes. My PR can wait on this one

**Pull request merged**